react/Product.js

```diff
@@ -99,18 +105,28 @@ const getAllSellers = (items) => {
   return flat
 }
 
+const getSellerDefault = (sellers) => {
+  const seller = sellers.find((s) => s.sellerDefault)
+
+  return seller
+}
+
 const composeAggregateOffer = (
   product,
   currency,
-  { decimals, pricesWithTax }
+  { decimals, pricesWithTax, useSellerDefault }
 ) => {
   const items = product.items || []
   const allSellers = getAllSellers(items)
   const { low, high } = lowHighForSellers(allSellers, { pricesWithTax })
 
   const offersList = items
     .map((element) =>
-      parseSKUToOffer(element, currency, { decimals, pricesWithTax })
+      parseSKUToOffer(element, currency, {
+        decimals,
+        pricesWithTax,
+        useSellerDefault,
+      })
```

Comment:
I'm seeing that at line 121 we already get the 'low' seller. Can you send the seller to the `parseSKUToOffer` to avoid duplicated logic?